examples/exotica_examples/scripts/example_ik_eff_axis_alignment.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+import rospy
+import pyexotica as exo
+import numpy as np
+from numpy import array
+from numpy import matrix
+import math
+from pyexotica.publish_trajectory import *
+from time import sleep
+import task_map_py
+from sensor_msgs.msg import Joy, JointState
+from std_msgs.msg import Float64MultiArray as FloatArray
+
+rospy.init_node('example_ik_eff_axis_alignment_node')
```

Comment:
Move to ``if __name__ == "__main__"``- not the top of the file.

Reply:
Is there a specific reason for this? Or is it just for style?

Reply:
It's wrong - when you like to include the file it can cause crashes and hangs otherwise (e.g., multiplicate initialisations). You should _not_ initialise a rospy node unless when needed in the main part. While this technically is a standalone example, since it contains a class it may be used differently or included into a notebook or larger script. It is a common mistake when people pick up this habit and has lead to a few users reporting "stuff hangs/doesnt work".